po/fr.po

```diff
@@ -6386,7 +6386,7 @@ msgstr "position verticale"
 
 #: ../src/iop/borders.c:1043
 msgid ""
-"select the vertical position ratio relative to left or right click and type "
+"select the vertical position ratio relative to left or right-click and type "
```

Comment:
You cannot change the main English string here. This should be done in the code and cannot be done at this stage as we are in string freeze.